testing/e2e.sh

```diff
@@ -378,11 +379,56 @@ function test_nodejs_image() {
 function test_container_push() {
   cd "${ROOT}"
   clear_docker
-  docker run -d -p 5000:5000 --name registry registry:2
+  cid=$(docker run --rm -d -p 5000:5000 --name registry registry:2)
   bazel build tests/docker:push_test
   # run here file_test targets to verify test outputs of push_test
```

Comment:
can you remove this line (added it for guidance for the other PR but no longer needed)